content/en/blog/posts/2022-02-01-build-release-v0.8.0/index.md

```diff
@@ -0,0 +1,116 @@
+
+## Features
+
+The list of features is not long; however I can ensure that they are all interesting!
+
+### Array support in Parameters
+
+We introduced an extension to the **parameter** feature, by allowing users to define parameters in the form of a list. A list can be composed of values from secrets, configmaps or plain values.
+
+Our main driver was the support for [ARGS](https://docs.docker.com/engine/reference/builder/#using-arg-variables) in Dockerfiles. This allows users to further [customize](https://github.com/shipwright-io/sample-go/blob/main/docker-build-with-args/Dockerfile) their builds, by specifying variables that are available to the `RUN` command.
+
+In addition, being able to use primitive resources(_such as secrets and configmaps_) to store key-values, allows users to protect confidential data or to share data when defining parameters values in their `Builds` or `BuildRuns`.
+
+**Note**: For more details on this, please see the [docs](https://github.com/shipwright-io/build/blob/main/docs/buildstrategies.md#strategy-parameters).
+
+### Surfacing Errors in the BuildRun Status
+
+Surfacing errors from different containers can be a challenging task, not because of technicality, but rather the question of the best way to represent the state. In case of _failure_ or _success_ during execution, we surface the state under the `.status` subresource of a `BuildRun`.
+
+![status subresource](image001.png)
+_The photo highlights the .status.failureDetails. subresource of a BuildRun_
+
+In this release we concentrated on improving the state of errors that occur during the cloning of _git_ repositories, by introducing `.status.failureDetails` field. This provides further details on why `step-source-default` failed.
+
+_Now you do not need to worry if you have git misconfigurations in your `Builds`, we got you cover!_
+
+**Note**: For more details, please see the [docs](https://github.com/shipwright-io/build/blob/main/docs/buildrun.md#understanding-failed-buildruns).
+
+### Local Source Upload
+
+At Shipwright, we've spent a lot of time trying to figure out the best ways to simplify the experience when building container images. In this release we are introducing a new feature that dramatically improves it, we call it **Local Source Upload** .
+
+This feature allows users to build container images from their local source code, improving the developer experience and moving them closer to the inner dev loop(_single developer workflow_).
+
+![shp cli](image002.png)
+_The photo above illustrates the use of the new shp cli subcommand._
+
+Go ahead and give it a try! The feature is now available in the [v0.8.0](https://github.com/shipwright-io/cli/releases/tag/v0.8.0) cli, look for `shp build upload`!
+
+## Important Notes
+
+{{< alert type="success" title="Action Required" >}}If you previously used the go types with parameters, then you will need to make slight changes to adopt to the changed type structure.{{< /alert >}}
```

Comment:
question - should we include this in an end-user blog post? This is really only a concern for projects that import Shipwright go code as a library.

Reply:
I will remove it, good point.